api/src/main/java/com/codeforcommunity/dto/team/GetAllTeamsResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.codeforcommunity.dto.team;
+
+import java.util.List;
+
+public class GetAllTeamsResponse {
+  private List<TeamSummary> teams;
+  private int rowCount;
```

Comment:
Can we call this something else? Is there a standard name we have for this field? @liammoyn

Reply:
It's the number of teams being returned, rowCount is what we've used in the past I believe